sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporter.java

```diff
@@ -51,6 +56,10 @@ public static void customize(AutoConfigurationCustomizer autoConfigurationCustom
      */
     public static void customize(AutoConfigurationCustomizer autoConfigurationCustomizer,
         AzureMonitorExporterOptions exporterOptions) {
+        if (customized) {
+            LOGGER.warning("You have already customized the OpenTelemetry auto-configuration for Azure.");
```

Comment:
It could be:
```
 throw LOGGER.logExceptionAsError(new IllegalStateException(
                "You have already customized the OpenTelemetry auto-configuration for Azure."));
```

But I'd rather not block the user if he uses the `customize` method twice with a valid use case that we haven't thought of.